src/main/java/org/elasticsearch/action/admin/cluster/health/ClusterHealthResponse.java

```diff
@@ -220,6 +234,11 @@ public void readFrom(StreamInput in) throws IOException {
                 validationFailures.add(in.readString());
             }
         }
+
```

Comment:
The tag on PR says that it goes only to 2.0.0. Is this PR going to be back-ported to v1.6.0 as well?